src/odb/src/db/dbProperty.cpp

```diff
@@ -86,30 +88,8 @@ bool _dbProperty::operator==(const _dbProperty& rhs) const
   if (_next != rhs._next)
     return false;
 
-  switch (_flags._type) {
-    case DB_STRING_PROP:
-      if (strcmp(_value._str_val, rhs._value._str_val) != 0)
-        return false;
-
-      break;
-
-    case DB_BOOL_PROP:
-      if (_value._bool_val != rhs._value._bool_val)
-        return false;
-
-      break;
-
-    case DB_INT_PROP:
-      if (_value._int_val != rhs._value._int_val)
-        return false;
-
-      break;
-
-    case DB_DOUBLE_PROP:
-      if (_value._double_val != rhs._value._double_val)
-        return false;
-
-      break;
+  if (_value == rhs._value) {
```

Comment:
Shouldn't this be != ?

In any case this can reduced to something like
return _value == rhs._value;

Reply:
I think I'd rather keep it as is. Feels like a deviation to just `return _value == rhs._value `